lab_4_fill_words_by_ngrams/main.py

```diff
@@ -125,6 +205,12 @@ def get_conversion_generator_type(self, generator_type: int) -> str:  # type: ig
         Returns:
             (str): Name of the generator.
         """
+        if generator_type == self.greedy:
+            return 'greedy generator'
+        if generator_type == self.top_p:
+            return 'top-p generator'
+        if generator_type == self.beam_search:
+            return 'beam search generator'
```

Comment:
can we rewrite it in such a way so that `None` cannot be returned?

Reply:
fixed